bmp388/registers.go

```diff
@@ -0,0 +1,84 @@
+// Package bmp388 provides a driver for Bosch's BMP388 digital temperature & pressure sensor.
+// The datasheet can be found here: https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bmp388-ds001.pdf
+package bmp388
+
+const ADDRESS byte = 0x77 // default I2C address
```

Comment:
The convention in Go is to not use all caps like here.

```suggestion
const Address byte = 0x77 // default I2C address
```

(also, the `byte` type is unnecessary but fine).